common/secret/token.ts

```diff
@@ -168,7 +175,18 @@ export async function verifyEmail(user: User) {
                 data: { verifiedAt: new Date(), verification: null },
                 where: { id: user.studentId },
             });
-            await Notification.actionTaken(user, 'student_registration_verified_email', {});
+            if (concreteNotification.sentAt) {
+                await Notification.actionTaken(user, 'student_registration_verified_email', {
+                    date: concreteNotification.sentAt
+                        .toLocaleDateString('de-DE', {
+                            day: 'numeric',
+                            month: 'long',
+                            year: 'numeric',
+                        })
+                        .replace('.', ''),
```

Comment:
Maybe it's better to use moment here? It should be configured properly already, and you have more control over the format.